netbout-client/src/test/java/com/netbout/client/RtInboxITCase.java

```diff
@@ -65,4 +67,39 @@ public void listsBouts() throws Exception {
         );
     }
 
+    /**
+     * RtInbox can search bouts.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void searchesBouts() throws Exception {
+        final Inbox inbox =
+            NbRule.get().aliases().iterate().iterator().next().inbox();
+        final Bout first = inbox.bout(inbox.start());
+        final String firsttitle = "bout1 title";
+        first.rename(firsttitle);
+        first.messages().post("hello");
+        final Bout second = inbox.bout(inbox.start());
+        final String secondtitle = "bout2 title";
+        second.rename(secondtitle);
+        second.messages().post("message with term");
+        final String thirdtitle = "bout title with term";
+        inbox.bout(inbox.start()).rename(thirdtitle);
+        final List<String> titles = new ArrayList<>(16);
+        for (final Bout bout : inbox.search("term")) {
+            titles.add(bout.title());
+        }
+        MatcherAssert.assertThat(
+            titles,
+            Matchers.hasItem(Matchers.equalTo(secondtitle))
```

Comment:
@CarlosGines you could use `hasItems` and match right away both found titles, instead of doing a separate assert